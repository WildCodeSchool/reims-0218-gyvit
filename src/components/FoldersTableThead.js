//TODO: create filters component for the folders page

import React from "react"
import { thead, tr, th } from "reactstrap"

const FoldersTableThead = () => (
  <thead>
    <tr>
      <th>Name</th>
      <th>Last update</th>
      <th>Users</th>
      <th />
    </tr>
  </thead>
)

export default FoldersTableThead
