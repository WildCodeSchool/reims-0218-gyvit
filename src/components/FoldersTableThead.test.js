import React from "react"
import ReactDOM from "react-dom"
import renderer from "react-test-renderer"
import FoldersTableThead from "./FoldersTableThead"

it("renders without crashing", () => {
  const table = document.createElement("table")
  ReactDOM.render(<FoldersTableThead />, table)
  ReactDOM.unmountComponentAtNode(table)
})

it("renders link html", () => {
  const tree = renderer.create(<FoldersTableThead />).toJSON()
  expect(tree).toMatchSnapshot()
})
