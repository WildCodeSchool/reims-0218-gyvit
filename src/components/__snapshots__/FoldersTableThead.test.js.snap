// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders link html 1`] = `
<thead>
  <tr>
    <th>
      Name
    </th>
    <th>
      Last update
    </th>
    <th>
      Users
    </th>
    <th />
  </tr>
</thead>
`;
